docs/openapi.yml

```diff
@@ -0,0 +1,547 @@
+openapi: 3.0.1
+info:
+  title: Marquez 
+  version: 1.0.0
+  description: The Marquez API is an interface for the collection, aggregation, and visualization
+    of a data ecosystems metadata.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: http://localhost:5000/api/v1
+    description: Local API server
+paths:
+  /namespaces/{namespace}:
+    parameters:
+      - $ref: '#/components/parameters/namespace'
+    put:
+      summary: Create a namespace
+      description: Creates a new namespace object. A namespace enables the contextual grouping of related
+        jobs and datasets. Namespaces must contain only letters (`a-z`, `A-Z`), numbers (`0-9`), or
+        underscores (`_`). A namespace is case-sensitive with a maximum length of `1024` characters.
+        Note jobs and datasets will be unique within a namespace, but not across namespaces.
+      tags:
+        - Namespaces
+      required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/namespaceReqBody'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/namespaceResBody'
+  /namespaces:
+    get:
+      summary: List all namespaces
+      description: Returns a list of namespaces.
+      tags:
+        - Namespaces
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/namespaces'
+ 
+  /namespaces/{namespace}/jobs/{job}:
```

Comment:
Add-on work for me to add the endpoint for getting info about a specific namespace based on its name.